saleor/thumbnail/views.py

```diff
@@ -85,4 +87,10 @@ def handle_thumbnail(
     thumbnail.image.save(thumbnail_file_name, thumbnail_file)
     thumbnail.save()
 
+    # set additional `instance` attribute, to easily get instance data
+    # for ThumbnailCreated subscription type
+    setattr(thumbnail, "instance", instance)
```

Comment:
Is it acceptable?

Reply:
As long as the comment explains it I think it is fine.